app/libs/utils.py

```diff
@@ -11,3 +11,12 @@ def convert_tx_id(tx_id):
 class ObjectFromDict:
     def __init__(self, properties):
         self.__dict__ = properties
+
+
+def get_answer(schema, answer_store, answer_id, list_item_id=None):
```

Comment:
Feel free to suggest what this method should be called and where this should live?. Maybe a new `answer_store_helper.py`?

Reply:
I've been looking at this and think it might be clearer if it only returned the answer tuple to lookup.

Reply:
This method has been removed. Is partly found in `rules.py`, not sure where it lives.